app/views/filters/displayConditions.scala.html

```diff
@@ -1,22 +1,11 @@
 @import org.incal.core.FilterCondition
+@import org.ada.web.util.{conditionTypeToReadableOperator, valueOrUndefined}
 
 @(conditions: Traversable[FilterCondition])
 
-@valueOrUndefined(condition: FilterCondition) = @{
```

Comment:
Good catch! Didn't realize `valueOrUndefined` was at two places. Glad you put it into utils.